src/plugins/data/public/antlr/shared/constants.ts

```diff
@@ -167,26 +167,67 @@ export const testingIndex = ({
 } as unknown) as IndexPattern;
 
 export const booleanOperatorSuggestions = [
-  { text: 'or', type: 17 },
-  { text: 'and', type: 17 },
+  { end: -1, start: -1, text: 'or', type: 17 },
+  { end: -1, start: -1, text: 'and', type: 17 },
 ];
 
-export const notOperatorSuggestion = { text: 'not', type: 17 };
+export const notOperatorSuggestion = { text: 'not', type: 17, end: -1, start: -1 };
 
-export const fieldNameSuggestions: Array<{ text: string; type: number; insertText?: string }> = [
-  { text: 'Carrier', type: 3, insertText: 'Carrier: ' },
-  { text: 'DestCityName', type: 3, insertText: 'DestCityName: ' },
-  { text: 'DestCountry', type: 3, insertText: 'DestCountry: ' },
-  { text: 'DestWeather', type: 3, insertText: 'DestWeather: ' },
-  { text: 'DistanceMiles', type: 3, insertText: 'DistanceMiles: ' },
-  { text: 'FlightDelay', type: 3, insertText: 'FlightDelay: ' },
-  { text: 'FlightNum', type: 3, insertText: 'FlightNum: ' },
-  { text: 'OriginWeather', type: 3, insertText: 'OriginWeather: ' },
-  { text: '_id', type: 3, insertText: '_id: ' },
-  { text: '_index', type: 3, insertText: '_index: ' },
-  { text: '_score', type: 3, insertText: '_score: ' },
-  { text: '_source', type: 3, insertText: '_source: ' },
-  { text: '_type', type: 3, insertText: '_type: ' },
+export const fieldNameSuggestions: Array<{
+  text: string;
+  type: number;
+  insertText?: string;
+  end: number;
+  start: number;
+}> = [
+  { text: 'Carrier', type: 3, insertText: 'Carrier: ', end: -1, start: -1 },
+  { text: 'DestCityName', type: 3, insertText: 'DestCityName: ', end: -1, start: -1 },
+  { text: 'DestCountry', type: 3, insertText: 'DestCountry: ', end: -1, start: -1 },
+  { text: 'DestWeather', type: 3, insertText: 'DestWeather: ', end: -1, start: -1 },
+  { text: 'DistanceMiles', type: 3, insertText: 'DistanceMiles: ', end: -1, start: -1 },
+  { text: 'FlightDelay', type: 3, insertText: 'FlightDelay: ', end: -1, start: -1 },
+  { text: 'FlightNum', type: 3, insertText: 'FlightNum: ', end: -1, start: -1 },
+  { text: 'OriginWeather', type: 3, insertText: 'OriginWeather: ', end: -1, start: -1 },
+  { text: '_id', type: 3, insertText: '_id: ', end: -1, start: -1 },
+  { text: '_index', type: 3, insertText: '_index: ', end: -1, start: -1 },
+  { text: '_score', type: 3, insertText: '_score: ', end: -1, start: -1 },
+  { text: '_source', type: 3, insertText: '_source: ', end: -1, start: -1 },
+  { text: '_type', type: 3, insertText: '_type: ', end: -1, start: -1 },
```

Comment:
should constants used for test be in a *.test.ts file?

Reply:
My assumption was that since the file didn't contain any tests, it shouldn't have that extension. I couldn't find any precedence for constants being used exclusively for tests, do you know if there is another way of handling these?

Reply:
ah @joshuali925 similar comment https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7593/files#r1732297731

you can use `mock` or just include this in the test.

exporting public fieldNameSuggestions could get confusing for other plugins since now any plugin can access it and this will get compiled down with the release distribution.

Reply:
that makes sense, i'll move everything here into those tests